apps/dashboard/app/views/projects/_form.html.erb

```diff
@@ -18,8 +18,17 @@
 
 <div class='card'>
 <h5 class='card-header'>Project Details</h5>
+  <%- unless @templates.to_a.empty? -%>
+    <div class="card">
+      <div class="card-body">
+        <div class="field">
+          <%= form.select(:template, @templates) %>
```

Comment:
We likely need to pass things to the label like `h3` to make the size of the text bigger to give it prominence. 

My concern is that it's easy to miss. You've moved it yes, but is it more prominent visually - I can't say.

It should also likely be `required` here too so that we reinforce the fact that we're making a project from a template.